### PR TITLE
style(web): Add some styling css to news and profile

### PR DESCRIPTION
**Pull request by hannaand**
Added basic styling 

## Tasks
- [ ] Fix styling to news component 
- [ ] Fix styling to news-detail component

### CODE REVIEW

File: src/Intranet.Web/Intranet.Web/ClientApp/app/components/app/app.component.css [hunk 1]
```diff
@@ -1,6 +1,18 @@
...
     /* On small screens, the nav menu spans the full width of the screen. Leave a space for it. */
     .body-content {
         padding-top: 50px;
+        
     }
+    h1 {
+		font-family: $roboto;
+    }	
 }
+
```
Comment (OskarKlintrot) [inline src/Intranet.Web/Intranet.Web/ClientApp/app/components/app/app.component.css]: I'm not sure all this whitespace is needed 😜

File: src/Intranet.Web/Intranet.Web/ClientApp/app/shared/pipes/truncate.ts [hunk 2]
```diff
@@ -0,0 +1,15 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'limitTo'
+})
+export class TruncatePipe {
+  transform(value: string, args: string) : string {
+    // let limit = args.length > 0 ? parseInt(args[0], 10) : 10;
+    // let trail = args.length > 1 ? args[1] : '...';
```
Comment (OskarKlintrot) [inline src/Intranet.Web/Intranet.Web/ClientApp/app/shared/pipes/truncate.ts]: Please remove dead code.

File: src/Intranet.Web/Intranet.Web/ClientApp/app/components/app/app.component.css (hunk 1 above)
Comment (hannaand) [inline src/Intranet.Web/Intranet.Web/ClientApp/app/components/app/app.component.css]: fixed

File: src/Intranet.Web/Intranet.Web/ClientApp/app/shared/pipes/truncate.ts (hunk 2 above)
Comment (hannaand) [inline src/Intranet.Web/Intranet.Web/ClientApp/app/shared/pipes/truncate.ts]: fixed

**Pull request merged**